packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -91,7 +103,7 @@ export default class MainProcess {
       path.join(__dirname, 'sharedProcess.js'),
       [`--runtimeSettingsJson=${JSON.stringify(this.settings)}`],
       {
-        stdio: 'inherit',
+        stdio: 'pipe',
```

Comment:
I was curious if this is going to have any bearing on logs but it looks like they're still collected properly.